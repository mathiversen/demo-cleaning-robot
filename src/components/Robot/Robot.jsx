import React from 'react'

import { css } from 'goober'

export default () => {
    let className = css`
        position: absolute;
        fill: none;
        stroke: white;
        stroke-width: 10px;
        stroke-location: inside;
        width: 90%;
        top: 50%;
        left: 50%;
        transform: translate(-50%, -50%);
    `

    return (
        <svg
            className={className}
            viewBox="0 0 100 100"
            xmlns="http://www.w3.org/2000/svg"
        >
            <circle cx="50" cy="50" r="40" />
        </svg>
    )
}
