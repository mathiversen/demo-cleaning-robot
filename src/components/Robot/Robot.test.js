import React from 'react'
import Robot from './Robot'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
    const tree = renderer.create(<Robot />).toJSON()
    expect(tree).toMatchSnapshot()
})
