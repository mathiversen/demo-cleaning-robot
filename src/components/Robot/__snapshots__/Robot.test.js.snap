// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<svg
  className="go2406169029"
  viewBox="0 0 100 100"
  xmlns="http://www.w3.org/2000/svg"
>
  <circle
    cx="50"
    cy="50"
    r="40"
  />
</svg>
`;
